### PR TITLE
Add footer

### PR DESCRIPTION
**Pull request by baomastr**

### CODE REVIEW

File: src/components/Footer/Footer.module.css [hunk 1]
```diff
@@ -1,3 +1,98 @@
...
+
+.footerNavLinksWrapper {
+    display: inline-flex;
+    margin-left: auto;
+    padding-top: 24px;
+    padding-bottom: 24px;
+
+    @media (min-width: 840px) {
+        padding-top: 48px;
+        padding-bottom: 48px;
```
Comment (therealemjy) [inline src/components/Footer/Footer.module.css:34]: Same comment as previously: I'd suggest using a base spacing from config.

File: src/components/Footer/Footer.module.css [hunk 2]
```diff
@@ -1,3 +1,98 @@
...
+        margin-left: 24px;
+    }
+}
+.copyWrapper {
+    display: flex;
+    align-items: center;
+    width: 100%;
+
+    @media (min-width: 640px) {
+        border-top: 1px solid #2A2B36;
```
Comment (therealemjy) [inline src/components/Footer/Footer.module.css:83]: Same thing about the config for the color

File: src/components/Footer/SocialLinks.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    href: "TODO3",
+  },
+  {
+    icon: <IconGh className={s.socialIcon} />,
+    href: "TODO4",
+  },
+]
+
+const SocialLinks: React.FC<ISocialLinksProps> = ({ className }) => {
+  return (
```
Comment (therealemjy) [inline src/components/Footer/SocialLinks.tsx:33]: You can use an implicit return

File: src/components/NavigationLinks/NavigationLinks.tsx [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    text: "Markets",
+  },
+  {
+    href: "TODO",
+    text: "Docs",
+  },
+]
+
+const NavigationLinks: React.FC<INavigationLinksProps> = ({ classNames }) => {
+  return (
```
Comment (therealemjy) [inline src/components/NavigationLinks/NavigationLinks.tsx:26]: Implicit return

**Comment by therealemjy** [comment]
Hey man,
I only commented on a few things, but I have the same comment as in your previous PR: is suggest storing your colors and base spacing in a config that you then import. If you want to use pure CSS you can use css variables.

File: src/components/Footer/Footer.module.css (hunk 1 above)
Comment (baomastr) [reply src/components/Footer/Footer.module.css:34]: don't think it makes sense in this project

**Comment by baomastr** [comment]
> Hey man, I only commented on a few things, but I have the same comment as in your previous PR: is suggest storing your colors and base spacing in a config that you then import. If you want to use pure CSS you can use css variables.

I didn't use variables in order of speed up. 
Going to move colors after finishing of mvp version. And don't think, that padding/margin has to be used with variables for current project.

**Pull request merged**